src/snmalloc/pal/pal.h

```diff
@@ -159,9 +159,7 @@ namespace snmalloc
    * The following are supported as arguments:
    *
    *  - Characters (`char`), printed verbatim.
-   *  - Strings (anything convertible to `std::string_view`), typically string
```

Comment:
As we've lost the other types that can flow into this, I am left wondering if we actually use this append function?

Reply:
As I guess I'm wondering if we should inline the functionality into append?

Reply:
Sorry, was busy on other stuffs. Inling should work except the two definitions will run into ambiguity.